.CI/create_feedstocks.py

```diff
@@ -254,10 +252,24 @@ def write_token(name, token):
                 team.url + "/memberships/" + new_member
             )
 
+    # Update status based on remote
+    subprocess.check_call(['git', 'checkout', os.environ.get('TRAVIS_BRANCH')])
+    subprocess.check_call(['git', 'pull', '--rebase', '--autostash'])
```

Comment:
That in turn could fail because a stash could be empty and popping an empty stash is an error. Tried to fix with PR ( https://github.com/conda-forge/staged-recipes/pull/1472 ).